l10n_it_edi_related_document/models/account_move.py

```diff
@@ -68,6 +68,86 @@ class AccountMove(models.Model):
         "account.move.related_document", "invoice_id", copy=False
     )
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            related_fields = [
+                "l10n_it_cig",
+                "l10n_it_cup",
+                "l10n_it_origin_document_type",
+                "l10n_it_origin_document_name",
+                "l10n_it_origin_document_date",
+            ]
+            if not any(vals.get(rf) for rf in related_fields):
+                continue
+
+            related_type = vals.get("l10n_it_origin_document_type")
+            if related_type == "purchase_order":
+                related_type = "order"
+
+            related_vals = {
+                "type": related_type,
+                "name": vals.get("l10n_it_origin_document_name"),
+                "date": vals.get("l10n_it_origin_document_date"),
+                "cup": vals.get("l10n_it_cup"),
+                "cig": vals.get("l10n_it_cig"),
+            }
+
+            if related_vals:
+                vals["related_document_ids"].append(Command.create(related_vals))
```

Comment:
Qui se 'related_document_ids' manca, va in errore
vals["related_document_ids"].append(Command.create(related_vals))
    ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'related_document_ids'